lib/zhr_devs/identity_management/event_handler.ex

```diff
@@ -0,0 +1,39 @@
+defmodule ZhrDevs.IdentityManagement.EventHandler do
+  @moduledoc """
+  This module is responsible for handling events.
+  Usually used to perform a side effect, such as sending an email or updating a read model.
```

Comment:
I don't understand what a "read model" is. I could learn, but copy pasta or a link to the doc would be cool!